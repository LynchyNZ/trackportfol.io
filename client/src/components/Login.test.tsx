import React from 'react';
import { shallow } from 'enzyme';
import LoginComponent from './Login';

describe('Login', () => {
  it('renders the Login component', () => {
    expect(LoginComponent).toMatchSnapshot();
  });

  it('displays the Login title (Sign in)', () => {
    const result = shallow(<LoginComponent />).contains('Sign in');
    expect(result).toBeTruthy();
  });
});
