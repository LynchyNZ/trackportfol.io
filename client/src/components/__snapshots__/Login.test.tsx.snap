// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders the Login component 1`] = `[Function]`;
